src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -66,10 +75,28 @@ protected void initWithJSon(String... jsonFiles) {
         createEmbeddingsDirectory();
     }
 
+    protected void initWithProperties(String... propertyFiles) {
+        if (propertyFiles != null) {
```

Comment:
as this is for test only I assume we can skip this condition and believe nobody is going to pass here `null` element ")

Reply:
cool